README.md

```diff
@@ -45,6 +45,8 @@ All of our examples can be found here in the form of
 Report bugs or feature requests using the
 [TensorFlow Quantum issue tracker](https://github.com/tensorflow/quantum/issues).
 
+We also have a [Stack Overflow tag](https://stackoverflow.com/questions/tagged/tensorflow-quantum).
```

Comment:
Maybe, "For more general advice, we also have ..."

Reply:
Added. PTAL.